punchcut/src/source/glyf/outline.rs

```diff
@@ -0,0 +1,165 @@
+use super::Point;
+use crate::PathSink;
+
+/// TrueType outline.
+#[derive(Default, PartialEq, Eq, Debug)]
+pub struct Outline {
+    /// Set of points that define the shape of the outline.
+    pub points: Vec<Point>,
+    /// Set of tags (one per point).
+    pub tags: Vec<u8>,
+    /// Index of the end points for each contour in the outline.
+    pub contours: Vec<u16>,
+    /// True if the scaler applied a scale, in which case the points are in
+    /// 26.6 fixed point format. Otherwise, they are in integral font units.
+    pub is_scaled: bool,
```

Comment:
Can wemodel this explicitly instead? Spitballing, make two types, make Point<T>, and then hold an enum here that is either `ScaledPoints(Vec<Point<26.6 guage fixed point>>)` or `ScaledPoints(Vec<Point<FontUnit>>)`

Reply:
I'd rather not have dynamic allocations embedded in an enum. One change I had considered here is to always return F26Dot6 values since the precision is more than enough to handle FWord and loading glyphs without a scale is likely fairly rare.

Reply:
The points field is now `Vec<F26Dot6>` regardless of scaling mode and the `is_scaled` field has been removed.